scripts/generate-openapi-script.sh

```diff
@@ -8,11 +8,14 @@ GENERATOR_VERSION="3.3.4"
 BASE_PATH="https://raw.githubusercontent.com/dockstore/dockstore/$npm_package_config_webservice_version"
 # DOCKSTORE-2428 - demo how to add new workflow language, generate from local copy of swagger
 # BASE_PATH="../dockstore"
+#CIRCLE_CI_PATH="https://3855-33383826-gh.circle-artifacts.com/0/tmp/artifacts"
```

Comment:
Intentionally left in? 
If so, might need to code comment `3855-33383826` (is that unique to "our" repo on circle?)

Reply:
It's intentionally left in for when I or others need to use a JAR for CircleCI.  The URL is apparent when you already uploaded a JAR via CircleCI

Reply:
Sounds good, add that as a code comment linking to https://wiki.oicr.on.ca/display/DOC/Dockstore+Releases#DockstoreReleases-PreDeploy and add a screenshot of where to look in CircleCI, thanks!